candidate_base.go

```diff
@@ -206,57 +205,53 @@ func (c *candidateBase) start(a *Agent, conn net.PacketConn, initializedCh <-cha
 }
 
 func (c *candidateBase) recvLoop(initializedCh <-chan struct{}) {
-	defer func() {
-		close(c.closedCh)
-	}()
+	defer close(c.closedCh)
 
 	select {
 	case <-initializedCh:
 	case <-c.closeCh:
 		return
 	}
 
-	log := c.agent().log
-	buffer := make([]byte, receiveMTU)
+	buf := make([]byte, receiveMTU)
 	for {
-		n, srcAddr, err := c.conn.ReadFrom(buffer)
+		n, sAddr, err := c.conn.ReadFrom(buf)
```

Comment:
`srcAddr` is kind of common across the Pion code bases. Happy to change if `sAddr` is a common pattern though!


```
sean@SeanLaptop:~/go/src/github.com/pion$ rg -l srcAddr
demo-conference/turn/main.go
webrtc/examples/vnet/show-network-usage/main.go
turn/client_test.go
turn/internal/server/turn_test.go
turn/internal/server/server.go
turn/internal/allocation/five_tuple_test.go
turn/internal/allocation/allocation.go
turn/examples/turn-server/tcp/main.go
turn/examples/turn-server/simple/main.go
turn/examples/turn-server/port-range/main.go
turn/lt_cred.go
turn/examples/turn-server/add-software-attribute/main.go
turn/examples/turn-server/log/main.go
turn/server_config.go
turn/server_test.go
turn/examples/turn-server/tls/main.go
ion-sfu/pkg/sfu/sfu.go
ion-sfu/pkg/sfu/turn.go
ice/connectivity_vnet_test.go
ice/candidate_relay_test.go
ice/candidate_base.go
transport/vnet/net_native_test.go
transport/vnet/conn_test.go
transport/vnet/net_test.go
transport/vnet/conn.go
transport/vnet/nat.go
transport/vnet/chunk.go
```

Reply:
Agreed. I renamed it to `srcAddr` in pion/ice